tests/ast/data/RelationalGroupedDataFrame.test

```diff
@@ -45,9 +45,9 @@ res8 = df.group_by("a").count()
 
 df = session.create_dataframe([("SF", 21.0), ("SF", 17.5), ("SF", 24.0), ("NY", 30.9), ("NY", 33.6)], schema=["location", "temp_c"])
 
```

Comment:
is the DoubleType() -> FloatType() change correct?

Reply:
This looks potentially a bug or difference in how local testing mode works.

For local testing, uses _extract_schema_and_data_from_pandas_df which maps to DoubleType while for non-local testing, uses the mapping PYTHON_TO_SNOW_TYPE_MAPPINGS (type_utils.py) to map 'float' -> 'FloatType'.